rainier-core/src/main/scala/com/stripe/rainier/repl/package.scala

```diff
@@ -177,5 +178,36 @@ package object repl {
   private def leftPad(s: String, len: Int, elem: Char): String =
     s.reverse.padTo(len, elem).reverse
 
+  def compare(models: (String, RandomVariable[_])*): Unit = {
+    //hardcoding the sampling parameters for now; we really need to refactor the sampling API to improve this
```

Comment:
It's pretty easy to add the version that takes sampler parameters to mirror `sample`. Maybe worth it given how often we've had to tune the sampler in the book.